### ISSUE TITLE
details list is missing aria-rowcount, aria-rowindex, and aria-colindex properties

### ISSUE THREAD

**Issue by petdun2519** [OfficeDev/office-ui-fabric-react#1666]
### Bug Report

- __Package version(s)__: (fill this out)
- __Browser and OS versions__: (fill this out if relevant)

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low) 

Products/sites affected: (if applicable)

#### Describe the issue:
There are some accessibility properties missing from details list grid

#### Actual behavior:
These properties are not present

#### Expected behavior:

These properties would be present

**Comment by ajaypantangi** [comment]
Seems like this is fixed in [PR](https://github.com/OfficeDev/office-ui-fabric-react/pull/1667), @cschleiden could you please close this issue? Thanks :)
